apps/prairielearn/python/prairielearn.py

```diff
@@ -1176,7 +1176,7 @@ def string_fraction_to_number(a_sub, allow_fractions=True, allow_complex=True):
     return (value, data)
 
 
-def string_to_2darray(s, allow_complex=True):
+def string_to_2darray(s, *, allow_complex=True):
```

Comment:
Need to check `string_to_2darray` callsites outside of our codebase.

Reply:
There are no active usages of this without named arguments.